networkit/cpp/algebraic/test/MatricesGTest.cpp

```diff
@@ -120,11 +120,23 @@ class MatricesGTest : public testing::Test {
     template <class Matrix>
     void testParallelForNonZeroElementsInRowOrder();
 
+    template <class Matrix>
+    void testMatrixToGraph();
+
     // TODO: Test other matrix classes
 
     // TODO: Test mmT multiplication, etc.!
 };
 
+template <class Matrix>
+void MatricesGTest::testMatrixToGraph() {
+
+    Matrix mat = Matrix::adjacencyMatrix(METISGraphReader{}.read("input/power.graph"));
+    Graph G = MatrixTools::matrixToGraph(mat);
+
+    EXPECT_EQ(G.numberOfEdges(), 6594);
```

Comment:
What about testing also the number of nodes and that the produced edges are correct? Further, which kind of graph do we expect? Should be weighted/unweighted, directed/undirected? I think these properties should be tested.